.NET/Microsoft.Recognizers.Text.DateTime/Dutch/Parsers/DutchSetParserConfiguration.cs

```diff
@@ -0,0 +1,137 @@
+
+        public IDateTimeParser TimeParser { get; }
+
+        public IDateExtractor DateExtractor { get; }
+
+        public IDateTimeParser DateParser { get; }
+
+        public IDateTimeExtractor DateTimeExtractor { get; }
+
+        public IDateTimeParser DateTimeParser { get; }
+
+        public IDateTimeExtractor DatePeriodExtractor { get; }
+
+        public IDateTimeParser DatePeriodParser { get; }
+
+        public IDateTimeExtractor TimePeriodExtractor { get; }
+
+        public IDateTimeParser TimePeriodParser { get; }
+
+        public IDateTimeExtractor DateTimePeriodExtractor { get; }
+
+        public IDateTimeParser DateTimePeriodParser { get; }
+
+        public IImmutableDictionary<string, string> UnitMap { get; }
+
+        public Regex EachPrefixRegex { get; }
+
+        public Regex PeriodicRegex { get; }
+
+        public Regex EachUnitRegex { get; }
+
+        public Regex EachDayRegex { get; }
+
+        public Regex SetWeekDayRegex { get; }
+
+        public Regex SetEachRegex { get; }
+
+        public DutchSetParserConfiguration(ICommonDateTimeParserConfiguration config) : base(config)
+        {
+            DurationExtractor = config.DurationExtractor;
+            TimeExtractor = config.TimeExtractor;
+            DateExtractor = config.DateExtractor;
+            DateTimeExtractor = config.DateTimeExtractor;
+            DatePeriodExtractor = config.DatePeriodExtractor;
+            TimePeriodExtractor = config.TimePeriodExtractor;
+            DateTimePeriodExtractor = config.DateTimePeriodExtractor;
+
+            DurationParser = config.DurationParser;
+            TimeParser = config.TimeParser;
+            DateParser = config.DateParser;
+            DateTimeParser = config.DateTimeParser;
+            DatePeriodParser = config.DatePeriodParser;
+            TimePeriodParser = config.TimePeriodParser;
+            DateTimePeriodParser = config.DateTimePeriodParser;
+            UnitMap = config.UnitMap;
+
+            EachPrefixRegex = DutchSetExtractorConfiguration.EachPrefixRegex;
+            PeriodicRegex = DutchSetExtractorConfiguration.PeriodicRegex;
+            EachUnitRegex = DutchSetExtractorConfiguration.EachUnitRegex;
+            EachDayRegex = DutchSetExtractorConfiguration.EachDayRegex;
+            SetWeekDayRegex = DutchSetExtractorConfiguration.SetWeekDayRegex;
+            SetEachRegex = DutchSetExtractorConfiguration.SetEachRegex;
+        }
+
+        public bool GetMatchedDailyTimex(string text, out string timex)
+        {
+            var trimmedText = text.Trim().ToLowerInvariant();
+            if (trimmedText.Equals("dagelijks"))
+            {
+                timex = "P1D";
+            }
+            else if (trimmedText.Equals("wekelijks"))
+            {
+                timex = "P1W";
+            }
+            else if (trimmedText.Equals("tweewekelijks"))
+            {
+                timex = "P2W";
+            }
+            else if (trimmedText.Equals("maandelijks"))
```

Comment:
Can you move these hard coded strings to the YAML resources? Some languages have this done partially already.

Reply:
Will do in a new PR, where I also fix the [EnglishSetParserConfiguration.cs](https://github.com/Microsoft/Recognizers-Text/blob/master/.NET/Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishSetParserConfiguration.cs). Most of my files are just a 1:1 copy of the English version at the moment.